src/frontend/src/binder/insert.rs

```diff
@@ -113,10 +116,54 @@ impl Binder {
             }
         };
 
+        let mut target_table_col_idxs: Vec<ColumnIdx> = vec![];
+        for query_column in &columns {
+            let column_name = &query_column.value; // value or real_value() ?
+            let mut col_exists = false;
+            for (col_idx, table_column) in table_source.columns.iter().enumerate() {
+                if *column_name.to_lowercase() == table_column.name.to_lowercase() {
+                    target_table_col_idxs.push(ColumnIdx::from(col_idx));
+                    col_exists = true;
+                    break;
```

Comment:
How about directly `continue 'outer` here?

Reply:
That is a nice idea. let me do that.